requirements_all.txt

```diff
@@ -1386,7 +1386,7 @@ praw==7.5.0
 prayer_times_calculator==0.0.6
 
 # homeassistant.components.progettihwsw
-progettihwsw==0.1.1
+progettihwsw==0.1.3
```

Comment:
This file is generated, what you need to change is the integration manifest.

Reply:
ok, i check this file